deny.toml

```diff
@@ -52,9 +52,7 @@ notice = "warn"
 # RUSTSEC-2021-0113 and RUSTSEC-2022-0006 are reported only because lightning crate depends on old regex and thread-local deps,
 # these will be removed after rust-lightning update https://github.com/KomodoPlatform/atomicDEX-API/issues/1298
 ignore = [
-    "RUSTSEC-2022-0013",
-    "RUSTSEC-2021-0113",
-    "RUSTSEC-2022-0006"
+    "RUSTSEC-2021-0113"
```

Comment:
Please remove RUSTSEC-2021-0113 and RUSTSEC-2022-0006 mentions from the above comment.

Reply:
@artemii235  When we will update `metrics-util` dep as well? We only use it in `/mm2src/common/mm_metrics/native.rs`, shouldn't be hard to update and get rid of `RUSTSEC-2021-0113`. I see the vulnerability has been fixed long time ago [here](https://github.com/metrics-rs/metrics/issues/190).

Reply:
They changed API significantly in newer versions, so updating it is not that easy (I've already tried it). The issue is already assigned to Sergey: https://github.com/KomodoPlatform/atomicDEX-API/issues/1299. We might not rush with it because possible race conditions in metrics is not critical for us.